lib/middleware/request_tracker.rb

```diff
@@ -124,6 +124,10 @@ def self.log_request(data)
       end
     end
 
+    if data[:browser_page_view] && !data[:is_crawler]
```

Comment:
I think this conditional doesn't match the conditionals when `page_view_anon_browser` is incremented especially when `SiteSetting.login_required` is `true`. For an anon user, we will end up triggering an event but the count would not be incremented.

A safer pattern might be to do what we do for `TopicsController.defer_topic_view` and just call a method to trigger an event right after `page_view_anon_browser` or `page_view_logged_in_browser` is incremented.